easybuild/easyconfigs/r/Ruby/Ruby-2.6.3-GCCcore-8.2.0.eb

```diff
@@ -0,0 +1,133 @@
+name = 'Ruby'
+version = '2.6.3'
+
+homepage = 'https://www.ruby-lang.org'
+description = """Ruby is a dynamic, open source programming language with 
+ a focus on simplicity and productivity. It has an elegant syntax that is 
+ natural to read and easy to write."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.2.0'}
+
+source_urls = ['http://cache.ruby-lang.org/pub/ruby/%(version_major_minor)s']
```

Comment:
https ?? or doesn't it work on that one?

Reply:
`https` does work, fixed in dbf3c21e5